src/pages/About/About.module.scss

```diff
@@ -32,7 +32,7 @@
 
       .title {
         @include medium(48px, Poppins, 24px);
-        color: #ea8033;
+        color: #F06500;
```

Comment:
check this, I think Nathan changed the orange theme. If that's the case, make the change in the index.scss file

Reply:
the orange in index.scss was correct, updated to use `$orange`